info/v1/container.go

```diff
@@ -415,6 +415,9 @@ type FsStats struct {
 	// Number of bytes available for non-root user.
 	Available uint64 `json:"available"`
 
+	// HasInodes when true, indicates that Inodes info will be available.
+	HasInodes bool `json:"has_inodes"`
```

Comment:
I was suggesting to move this to [ContainerSpec](https://github.com/google/cadvisor/blob/3977cb8bd1fb734369513336aa0c20a47330d2e4/info/v1/container.go#L44), or better yet, create a `FsSpec` struct in that file with this field, and link to that struct from the `ContainerSpec`.